en/concept/custom_mixer_payload.md

```diff
@@ -1,5 +1,10 @@
 # Custom Payload Mixers
 
+:::warning
```

Comment:
@bkueng This is basically the quickest way I can mark this as "do not use" without deleting it. 

What I need to understand is when you'd use it in practice. I THINK that this would be when you wanted to implement your own real dedicated command for controlling a "thing". So essentially say I want to add a gripper payload and I want to use a specific MAV_CMD to do that. I'd follow these instructions.

But the story still doesn't feel "complete" for that, firstly because it doesn't map through to RC (which you might also want). Secondly, if you're adding hardware you want an easy way to say "the gripper hardware is on this servo" so that a gripper message can magically map through to that servo. I think that means that we should *reserve* control group ids for specific purposes, or there is no way to know that the output has actually be used/assigned. 

Upshot, I don't understand how this would ever be useful. Can you explain, or perhaps we should delete it?

Reply:
I would just delete it for now, also because the implementation will change quite a bit within the next months or so (there will be a concept of 'functions', where in your example you would just add the gripper as a new function, make sure it's handled in the output driver library, and you're done. The user can then select the function via param).

> I think that means that we should reserve control group ids for specific purposes, or there is no way to know that the output has actually be used/assigned.

Yes that is correct.